spglm/varfuncs.py

```diff
@@ -114,9 +114,16 @@ def deriv(self, mu):
         """
         Derivative of the variance function v'(mu)
         """
-        from statsmodels.tools.numdiff import approx_fprime_cs, approx_fprime
+
+        ########################################################################
+        # `approx_fprime_cs` is imported by unused
+        from statsmodels.tools.numdiff import approx_fprime_cs  # noqa F401
```

Comment:
Do we need it at all?

Reply:
I'm not a maintainer here, but I think the `approx_fprime_cs()` should be used for maximum robustness. It looks like it was removed due to some kind of bug/compatability regression?

Reply:
So the problem here is that it [isn't covered in testing](https://github.com/pysal/spglm/actions/runs/6484842887/job/17609565228#step:6:52).

Reply:
But perhaps it may be better to leave as is for now and merge this PR since no functionality is changing? The issue is documented, so it can be further discussed in time. Thoughts @martinfleis @ljwolf ?

Reply:
fine with me